src/Microsoft.VisualBasic.Core/tests/ProjectDataTests.cs

```diff
@@ -48,5 +48,12 @@ public void ClearProjectError()
             Assert.Null(Information.Err().GetException());
             Assert.Equal(0, Information.Err().Erl);
         }
+
+        [Fact]
+        public void EndApp()
+        {
+            // Cannot invoke EndApp because that will exit the process. Simply verify the method exists.
```

Comment:
You can actually use our test utility function: just add `using Microsoft.DotNet.RemoteExecutor;` then wrap the code like 

```C#
RemoteExecutor.Invoke(() =>
     // call the method to exit the process
     throw new Exception("should not get here - if it does the test will fail");
);
```

Reply:
You can verify the exit code as well ..

Reply:
Great, thanks!

---
In reply to: [277344899](https://github.com/dotnet/corefx/pull/37085#discussion_r277344899) [](ancestors = 277344899)